frontend/apps/cloudstorage/webdavapi.lua

```diff
@@ -95,6 +96,11 @@ function WebDavApi:listFolder(address, user, pass, folder_path)
     socketutil:reset_timeout()
     if headers_request == nil then
         return nil
+    elseif headers_request < 200 or headers_request >= 300 then
+        -- got a response, but it wasn't a success (e.g. auth failure)
+        logger.dbg(headers_request)
+        logger.dbg(table.concat(sink))
+        return nil
```

Comment:
Unless we care about the distinction. Btw, logger supports tables.
```suggestion
        return
```

Reply:
The branch above did the same, FWIW.